test/transcript/TranscriptEvent.test.ts

```diff
@@ -60,10 +68,15 @@ describe('TranscriptEvent', () => {
     const dataMessage = makeTranscriptDataMessageFrom(
       TRANSCRIPT_EVENT_TEST_VECTORS.TRANSCRIPTION_STATUS_STARTED_WITH_MESSAGE
     );
-    const actualEvents = TranscriptEvent.from(dataMessage);
+    const actualEvents = TranscriptEventConverter.from(dataMessage);
 
     expect(actualEvents.length).to.equal(1);
-    expect(actualEvents[0].status.message).to.eql('detail message');
+    if (actualEvents[0] instanceof Transcript) {
+      assert.fail();
+      return;
+    }
```

Comment:
I'm curious if the inverted check — not `Status` — works?

Reply:
Good catch! This is actually the correct way to check. Updated.